
data_root_dir: /data/debug/online_ss_tests
data_set_name: 2022_06_30_00_17_59Z_r4

target_frame: map
